### PR TITLE
[Non-modular] Necroposting antag update

### PR DESCRIPTION
**Pull request by Stalkeros2**
# What this PR does
Adds loot drops to both Syndicate and revs, introduces new currency for revs (that has no icon in vendors for some reason and also has no mapped in vendors so you gotta replace syndies vendors with revs ones and then remove their TCs), adds SKAT-10 aka NT's SKAT and NT Mauler Mk2 aka NT's Mauler Mk2.

# Why it should be added to the game
More content good, makes rev and syndi waves actually viable now.

### CODE REVIEW

**Comment by Stalkeros2** [comment]
merg when

File: code/_core/obj/item/container/beaker/syringe.dm [hunk 1]
```diff
@@ -205,6 +206,57 @@
...
+
+	icon = I
+
+	return ..()
+//le shitcode below, read at your own risk
+/obj/item/container/syringe/medipen/click_self(var/mob/caller as mob,var/atom/object,location,control,params)
+	INTERACT_CHECK
+	INTERACT_DELAY(1)
+	injecting = TRUE
+	if (sealed == TRUE)
```
Comment (BurgerLUA) [inline code/_core/obj/item/container/beaker/syringe.dm:236]: This should be just ``if(sealed)`` as ``== TRUE`` is redundent

File: code/_core/obj/item/container/beaker/syringe.dm [hunk 2]
```diff
@@ -205,6 +206,57 @@
...
+	else
+		sealed = TRUE
+		caller.to_chat(span("notice","You put the cap back on."))
+		icon = icon(icon,"medipen_closed")
+	return
+
+/obj/item/container/syringe/medipen/can_inject(var/mob/caller,var/atom/target)
+	if (sealed == TRUE)
+		caller.to_chat(span("warning","Take the cap off before injecting!"))
+		return
```
Comment (BurgerLUA) [inline code/_core/obj/item/container/beaker/syringe.dm:249]: Should be return FALSE

File: code/_core/obj/item/container/beaker/syringe.dm [hunk 3]
```diff
@@ -205,6 +206,57 @@
...
+		caller.to_chat(span("notice","You uncap the injector."))
+		icon = icon(icon,"medipen_open")
+	else
+		sealed = TRUE
+		caller.to_chat(span("notice","You put the cap back on."))
+		icon = icon(icon,"medipen_closed")
+	return
+
+/obj/item/container/syringe/medipen/can_inject(var/mob/caller,var/atom/target)
+	if (sealed == TRUE)
```
Comment (BurgerLUA) [inline code/_core/obj/item/container/beaker/syringe.dm:247]: This should be just if(sealed) as == TRUE is redundent

File: code/_core/obj/item/container/beaker/syringe.dm [hunk 4]
```diff
@@ -205,6 +206,57 @@
...
+		caller.to_chat(span("warning","Take the cap off before injecting!"))
+		return
+	else
```
Comment (BurgerLUA) [inline code/_core/obj/item/container/beaker/syringe.dm:250]: else here is redundent.

File: code/_core/obj/item/container/beaker/syringe.dm [hunk 5]
```diff
@@ -205,6 +206,57 @@
...
+		.=..()
+
+/obj/item/container/syringe/medipen/inject(var/mob/caller,var/atom/object,var/amount=5)
+	if (sealed == TRUE)
```
Comment (BurgerLUA) [inline code/_core/obj/item/container/beaker/syringe.dm:254]: This should be just if(sealed) as == TRUE is redundent

File: code/_core/obj/item/container/beaker/syringe.dm [hunk 6]
```diff
@@ -205,6 +206,57 @@
...
+		caller.to_chat(span("warning","Take the cap off before injecting!"))
+		return
```
Comment (BurgerLUA) [inline code/_core/obj/item/container/beaker/syringe.dm:256]: should be return FALSE

File: code/_core/obj/item/container/beaker/syringe.dm [hunk 7]
```diff
@@ -205,6 +206,57 @@
...
+	else
```
Comment (BurgerLUA) [inline code/_core/obj/item/container/beaker/syringe.dm:257]: else here is redundent

**Review by BurgerLUA — CHANGES_REQUESTED** [review]

**Comment by Stalkeros2** [comment]
Les gooooooo.

File: code/_core/datum/status_effect/_status_effect.dm [hunk 8]
```diff
@@ -214,6 +214,20 @@
...
+	minimum = 200 // 20 seconds
+	maximum = 3 * 60 * 10 //5 minutes.
+
+/status_effect/stimmed/on_effect_added(var/mob/living/owner,var/atom/source,var/magnitude,var/duration,var/stealthy)
+	. = ..()
+	if(owner.health) owner.health.update_health(check_death=FALSE)
+	owner.remove_status_effect(STAMCRIT)
+	owner.remove_status_effect(CRIT)
+	owner.stamina_regen_delay = 0
+
```
[the remark this answers is not in the document]
Comment (Stalkeros2) [reply code/_core/datum/status_effect/_status_effect.dm:230]: :thinking:

**Comment by Stalkeros2** [comment]
AFK a little bit so it's gonna be delayed for a while. :pensive:

**Comment by jjpark-kb** [comment]
You will need to go through all of the files and resolve the conflicts.

File: code/_core/obj/item/clothing/head/helmet/hardsuit/russian.dm [hunk 9]
```diff
@@ -29,11 +29,34 @@
...
-		BLUNT = AP_CLUB,
-		PIERCE = AP_CLUB,
+		BLADE = AP_AXE*1.25,
+		BLUNT = AP_CLUB*1.25,
+		PIERCE = AP_CLUB*1.25,
+		LASER = AP_AXE,
+		ARCANE = -AP_GREATAXE*0.8,
+		HEAT = AP_AXE,
+		COLD = AP_AXE,
+		BOMB = AP_AXE
```
Comment (jjpark-kb) [inline code/_core/obj/item/clothing/head/helmet/hardsuit/russian.dm:39]: AP_[NAME] is no longer valid, please consult similar items for defense ratings.

File: code/_core/obj/item/clothing/head/helmet/hardsuit/russian.dm [hunk 10]
```diff
@@ -29,11 +29,34 @@
...
+	desc_extended = "Modified version of revolutionaries' SKAT-9 combat suit, featuring scarce NT paintjob and, obviously, changed IFFs."
+	rarity = RARITY_RARE
+
+	defense_rating = list(
+		BLADE = AP_AXE*1.25,
+		BLUNT = AP_CLUB*1.25,
+		PIERCE = AP_CLUB*1.25,
 		LASER = AP_AXE,
-		ARCANE = -AP_GREATAXE,
+		ARCANE = -AP_GREATAXE*0.8,
```
Comment (jjpark-kb) [inline code/_core/obj/item/clothing/head/helmet/hardsuit/russian.dm:59]: AP_[NAME] is no longer valid, please consult similar items for defense ratings.

File: code/_core/obj/item/clothing/head/helmet/hardsuit/syndicate.dm [hunk 11]
```diff
@@ -65,4 +65,25 @@
...
+		BLUNT = AP_CLUB,
+		PIERCE = AP_CLUB,
+		LASER = AP_CLUB,
+		ARCANE = -AP_CLUB,
+		HEAT = AP_AXE,
+		COLD = AP_AXE,
+		BOMB = AP_AXE,
+		BIO = AP_SWORD,
+		RAD = AP_SWORD,
+		PAIN = AP_CLUB
```
Comment (jjpark-kb) [inline code/_core/obj/item/clothing/head/helmet/hardsuit/syndicate.dm:88]: AP_[NAME] is no longer valid, please consult similar items for defense ratings.

File: code/_core/obj/item/clothing/overwear/hardsuit/russian.dm [hunk 12]
```diff
@@ -41,11 +41,11 @@
...
 	defense_rating = list(
-		BLADE = AP_AXE,
-		BLUNT = AP_CLUB,
-		PIERCE = AP_CLUB,
+		BLADE = AP_AXE*1.25,
+		BLUNT = AP_CLUB*1.25,
+		PIERCE = AP_CLUB*1.25,
 		LASER = AP_AXE,
-		ARCANE = -AP_GREATAXE,
+		ARCANE = -AP_GREATAXE*0.8,
```
Comment (jjpark-kb) [inline code/_core/obj/item/clothing/overwear/hardsuit/russian.dm:48]: AP_[NAME] is no longer valid, please consult similar items for defense ratings.

File: code/_core/obj/item/clothing/overwear/hardsuit/russian.dm [hunk 13]
```diff
@@ -58,4 +58,34 @@
...
+
+	defense_rating = list(
+		BLADE = AP_AXE*1.25,
+		BLUNT = AP_CLUB*1.25,
+		PIERCE = AP_CLUB*1.25,
+		LASER = AP_AXE,
+		ARCANE = -AP_GREATAXE*0.8,
+		HEAT = AP_AXE,
+		COLD = AP_AXE,
+		BOMB = AP_AXE
```
Comment (jjpark-kb) [inline code/_core/obj/item/clothing/overwear/hardsuit/russian.dm:81]: AP_[NAME] is no longer valid, please consult similar items for defense ratings.

File: code/_core/obj/item/clothing/overwear/hardsuit/syndicate.dm [hunk 14]
```diff
@@ -83,4 +83,34 @@
...
+		BLUNT = AP_CLUB,
+		PIERCE = AP_CLUB,
+		LASER = AP_CLUB,
+		ARCANE = -AP_CLUB,
+		HEAT = AP_AXE,
+		COLD = AP_AXE,
+		BOMB = AP_AXE,
+		BIO = AP_SWORD,
+		RAD = AP_SWORD,
+		PAIN = AP_CLUB
```
Comment (jjpark-kb) [inline code/_core/obj/item/clothing/overwear/hardsuit/syndicate.dm:106]: AP_[NAME] is no longer valid, please consult similar items for defense ratings.

**Comment by Stalkeros2** [comment]
Whew, I'm finally not dead.

**Comment by Stalkeros2** [comment]
LETS FUCKING GOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

**Comment by Stalkeros2** [comment]
Okay no I'd rather start a new one because this one is fucked beyond recognition.

**Pull request closed without merging**